tinyms/text/transforms.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2021 Huawei Technologies Co., Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ============================================================================
+import tinyms as ts
+
+from .. import vision
```

Comment:
Please use `from mindspore.dataset.transforms.c_transforms import *` instead, and then change `__all__` to append:
```python
common_trans = [
    'Compose',
    'Concatenate',
    'Duplicate',
    'Fill',
    'Mask',
    'OneHot',
    'PadEnd',
    'RandomApply',
    'RandomChoice',
    'Slice',
    'TypeCast',
    'Unique',
]
```